doc/en/api/1.0.0/gwcreload.yaml

```diff
@@ -0,0 +1,19 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Reloading
+  description: Reloading a GeoWebCache instance after changing configurations.
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /reload:
+    post:
+      operationId: reloadPost
+      summary: Reloads GWC settings/
+      description: Reloads the GeoWebCache settings after making changes to the configuration. Can include the the data parameter in request to reload the new settings from data.
```

Comment:
You mention a data parameter, but don't document it as a parameter. Can you add it?

Reply:
Added in some clarification.  Examples I saw used -d 'data' to reload the configuration based on that data setting.